### PR TITLE
Ashwalkers in Lavalands now use their proper tools

### PR DESCRIPTION
**Pull request by IgiariValkyr**
## About The Pull Request

Ashwalkers in Lavalands are known kleptomaniacs that continue to use human tools instead of the crafted tools by their tribe. Even the Ice Moon tribe uses the crafted tools. So this just brings them in line with each other.

lavalandsurgerybeforeafter
lavalandstoragebeforeafter

## How This Contributes To The Skyrat Roleplay Experience

Just brings the Lavalands Tribe to par with the Ice Moon Tribe in terms of appropriate tools.

## Changelog

:cl:
balance: Lavaland Ashwalkers now have Primative Tools instead of Human Tools
/:cl:

### CODE REVIEW

**Comment by IgiariValkyr** [comment]
oh fuck wait I accidently deleted the lizard blood bags

File: _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm [hunk 1]
```diff
@@ -1648,8 +1649,16 @@
 	dir = 4
 	},
 /obj/structure/table/bronze,
-/obj/item/storage/belt/utility/full,
```
Comment (Zonespace27) [inline _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm:1651]: add a subtype of utility belts with ashwalker tools

File: _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm [hunk 2]
```diff
@@ -924,27 +924,28 @@
...
-/obj/item/hemostat,
-/obj/item/retractor,
+/obj/structure/closet/crate/wooden,
+/obj/item/cautery/ashwalker,
+/obj/item/circular_saw/ashwalker,
+/obj/item/hemostat/ashwalker,
+/obj/item/retractor/ashwalker,
+/obj/item/scalpel/ashwalker,
+/obj/item/surgicaldrill/ashwalker,
+/obj/structure/stone_tile/cracked{
```
Comment (Zonespace27) [inline _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm:934]: these two tiles are stacked ontop of eachother

File: _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm [hunk 3]
```diff
@@ -924,27 +924,28 @@
 /turf/closed/indestructible/riveted/boss/see_through,
 /area/ruin/unpowered/ash_walkers)
 "FL" = (
-/obj/item/storage/medkit/regular,
-/obj/item/reagent_containers/blood/random,
-/obj/item/reagent_containers/blood/lizard,
-/obj/item/reagent_containers/blood/lizard,
-/obj/item/stack/sheet/cloth/ten,
```
Comment (Zonespace27) [inline _maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm:931]: intended to delete cloth?

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**